dev/local-environment/configurations/cardano/v-function-updated.script

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "PlutusScriptV2",
+    "description": "V-function that allows to release one token each minute since 2025-02-07 10:46 UTC (timestamp 1738925213000)",
```

Comment:
Can we use something round, like 1st Feb, to make calculations easier? After all humans are supposed to use this setup. If this is only for E2E, then it should be moved there.

Reply:
sure, I'll move it then